test.sh

```diff
@@ -63,9 +63,12 @@ SvrPerfOutfileArg=
 # execution, we can now specify a list of server-threads to execute,
 # using env-var as:
 #
-#   SERVER_NUM_THREADS="1 2 4 8" ./test.sh run-all-client-server-perf-tests
+#   L3_PERF_SERVER_NUM_THREADS="1 2 4 8" ./test.sh run-all-client-server-perf-tests
```

Comment:
Slight change in env-var: Use `L3_PERF` prefix, to namespace the env-vars used by this script.